src/features/swaps/SwappingScreen.tsx

```diff
@@ -35,10 +38,7 @@ const SwappingScreen = () => {
 
   const onReturn = useCallback(() => {
     // Reset Swap stack to first screen
-    navigation.reset({
-      index: 0,
-      routes: [{ name: 'Home' }],
-    })
+    navigation.goBack()
```

Comment:
Trying to think why it was a reset before, is there a deep link here or something where the app cant go back? Do we need a check for `navigation.canGoBack()` and if not do the reset? Why did this get changed?

Reply:
It was causing weird behavior because the navigator would need to load back. This makes it smoother